### PR TITLE
B 19938 int

### PR DESCRIPTION
**Pull request by antgmann**
## Summary

This merge simply adds a new text column to the MTO_Shipments database, titled "diversion_reason" to store the reasons for an MTO Shipment diversion.

## Verification Steps for the Author

These are to be checked by the author.
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### How to test

1. After checking out the branch, run a fresh migration to add the column to the table. Verify the column appears with it's description.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

File: migrations/app/schema/20240503123556_add_diversion_reason_to_mto_shipments.up.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- Adds new column to mto shipments table
+ALTER TABLE mto_shipments
+ADD COLUMN IF NOT EXISTS diversion_reason TEXT NULL;
+
+-- Comments on new column
+COMMENT ON COLUMN mto_shipments.actual_pro_gear_weight IS 'Stores the reason for a requested diversion.';
```
Comment (ajlusk) [inline migrations/app/schema/20240503123556_add_diversion_reason_to_mto_shipments.up.sql:6]: Looks like there's a copy and paste error here. Comment should be on the diversion_reason column.

Comment (antgmann) [reply migrations/app/schema/20240503123556_add_diversion_reason_to_mto_shipments.up.sql:6]: Yup you're right lol let me fix that

Comment (r-mettler) [reply migrations/app/schema/20240503123556_add_diversion_reason_to_mto_shipments.up.sql:6]: Good catch 😄

**Comment by r-mettler** [comment]
image
I see the field in the database.

File: migrations/app/schema/20240503123556_add_diversion_reason_to_mto_shipments.up.sql (hunk 1 above)
Comment (antgmann) [reply migrations/app/schema/20240503123556_add_diversion_reason_to_mto_shipments.up.sql]: Resolved in latest commit

**Review by r-mettler — APPROVED** [review]

**Review by MInthavongsay — APPROVED** [review]

**Pull request merged**